Gateway_v2/decoder.py

```diff
@@ -156,15 +156,15 @@ def sort_packet(self, data, timestamp):
       dataDict["press_pa"] = unpacked_data[5]
       dataDict["temp_c"] = unpacked_data[6]
       dataDict["humidity_centi_pct"] = unpacked_data[7]
-      dataDict["apogee_w_m2"] = unpacked_data[8]
+      dataDict["apogee_w_m2"] = unpacked_data[8] * 0.25
```

Comment:
Are these values from the datasheet? It would be good to add some comments or name the conversion factor to something descriptive.

The firmware team has had some pain from reading code without comments about the conversion factors...